core/src/subgraph/instance_manager.rs

```diff
@@ -534,6 +534,9 @@ where
                     // Revert the in-memory state:
                     // - Remove hosts for reverted dynamic data sources.
                     // - Clear the entity cache.
+                    //
+                    // Note that we do not currently revert the filters,
+                    // which is not ideal, but also not incorrect.
```

Comment:
It would be better to explain why that is not incorrect (I have no idea what exactly having these filters will lead to)